csv-pricing.md

```diff
@@ -0,0 +1,118 @@
+# CSV Pricing
+
+> **Note**: This feature is only oficially supported on Kubecost Enterprise plans and during free trials.
+
+The following resource documents the steps required to set up custom prices with a CSV pipeline. This feature allows for individual assets (e.g. nodes) to be supplied at unique prices. Common uses are for on-premise machines or for external enterprise discounts. 
+
+1. Provide a file path for your CSV pricing data in values.yaml. This path can reference a local PV or an S3 bucket. 
+
+    ```
+    pricingCsv:
+    enabled: true
+    location:
+    provider: "AWS|GCP"
+    region: "us-east-1" 
+    URI: < your-valid-file-URI >
+    csvAccessCredentials: pricing-schema-access-secret
+    ```
+    
+    S3 bucket access is supported with the format `s3://<bucket-name>/<key>`, for example 's3://kc-csv-test/pricing_schema.csv
+`. Otherwise, the CSV file will be read from the cost-analyzer PV, for example:
+
+    ```
```

Comment:
YAML syntax is very important. We need to be sure to retain the proper indentation and spacing.
The block should be formatted as follows:

``` yaml
pricingCsv:
  enabled: true
  location:
    URI: /var/kubecost-csv/custom-pricing.csv

extraVolumes:
- name: kubecost-csv
  configMap:
    name: csv-pricing

extraVolumeMounts:
- name: kubecost-csv
  mountPath: /var/kubecost-csv
```

@jessegoodier Please double check me on this.